test/foundry/apps.t.sol

```diff
@@ -749,4 +750,187 @@ contract FleekTest is Test {
+        );
+    }
+
+    function testFailDecreaseAccessPointScoreToMinusOne() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+        fleekContract.decreaseAccessPointScore("https://fleek_cloned.xyz");
+    }
+
+    function testDecreaseAccessPointScore() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+        fleekContract.increaseAccessPointScore("https://fleek_cloned.xyz");
+        fleekContract.decreaseAccessPointScore("https://fleek_cloned.xyz");
+        assertEq(
+            fleekContract.getAccessPointJSON("https://fleek_cloned.xyz"),
+            '{"tokenId":0,"score":0,"nameVerified":false,"contentVerified":false,"owner":"0x7fa9385be102ac3eac297483dd6233d62b3e1496"}'
+        );
+    }
+
+    function testAppAccessPoints() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+
+        string[] memory accessPointList = fleekContract.appAccessPoints(mint);
+        assertEq(accessPointList[0], "https://fleek_cloned.xyz");
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned_2.xyz");
+
+        accessPointList = fleekContract.appAccessPoints(mint);
+        assertEq(accessPointList[1], "https://fleek_cloned_2.xyz");
+    }
+
+    function testFailSetAccessPointNameVerifyWithUnknownIdentity() public {
+        uint256 mint = fleekContract.mint(
+            DEPLOYER,
+            "Foundry Test App",
+            "This is a test application submitted by foundry tests.",
+            "https://fleek.xyz",
+            "fleek_xyz",
+            "afff3f6",
+            "https://github.com/fleekxyz/non-fungible-apps"
+        );
+
+        assertEq(mint, 0);
+
+        fleekContract.addAccessPoint(0, "https://fleek_cloned.xyz");
+
+        vm.prank(address(0xb4c79daB8f259C7Aee6E5b2Aa729821864227e84));
```

Comment:
Maybe a comment above this to say it set's the sender to the provided address.All the other stuff was very well self describing but this wasn't clear so comment can help someone new that reads it.